tests/shared/nlu/training_data/test_training_data.py

```diff
@@ -295,6 +295,34 @@ def test_train_test_split(filepaths: List[Text]):
         )
 
 
+def test_number_of_examples_per_intent():
```

Comment:
would this test have failed before your change?

Reply:
yup, exactly! this test fails before the amendment to `nlu_examples`

Reply:
because `message_intent_two` would be counted? Maybe a comment or different names would help to show which should and shouldn't be counted? 

Also would it be possible to test this on some actual yaml training data files? e.g. from in the `data/` directory? Would be good to show how it fits into the bigger picture.

Reply:
Ok, yes, have amended names to reflect which is `nlu `and which `non_nlu `- haven't yet pushed changes.
I couldn't find an example in the data/ directory that has matching intents in `nlu.yml `and `stories.yml `or `rules.yml `files? Should I create my own subdirectory with some yml files - let's say test_number_of_nlu_examples?

Reply:
all done now 😄